Src/Couchbase/Couchbase.csproj

```diff
@@ -1,61 +1,53 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
+﻿<Project Sdk="Microsoft.NET.Sdk">

```

Comment:
Are there any changes in the csproj other than formatting? If just formatting, can we ignore them for this commit?

Reply:
Sure, I'll remove this change